.jshintrc

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
It's been on my system for so long it's hard not to just use it... Let me clean up the PR.